src/sagemaker/config/config_schema.py

```diff
@@ -639,6 +644,24 @@ def _simple_path(*args: str):
                                         },
                                     },
                                 },
+                                ESTIMATOR: {
+                                    TYPE: OBJECT,
+                                    ADDITIONAL_PROPERTIES: False,
+                                    PROPERTIES: {
+                                        DEBUG_HOOK_CONFIG: {
+                                            TYPE: "boolean",
+                                            "description": (
+                                                "Sets a boolean for `debugger_hook_config` of"
```

Comment:
Curious - where will the customer see this info?

Reply:
I think it will help customers to understand where this config will be applied. I am not sure if the description field is also utilized in case there are any ValidationErrors for this field during json schema validation.